dist/index.html

```diff
@@ -104,6 +104,7 @@ <h1>
           A neural network will try to guess what it’s seeing.<br />Make sure your sound is on.</p>
         <button class="btn play-btn landing__play-btn--js">Let's Play</button>
         <p class="view__landing__desktop-msg view__landing__desktop-msg--js">Emoji Scavenger Hunt is best experienced on phones.</p>
+        <p class="view__landing__age-msg view__landing__age-msg--js">By pressing the button above, I certify that I am over 13 years old.</p>
```

Comment:
Now that we merged that iOS change let's rebase this so this change doesn't show up here again

Reply:
Done.